mlflow/models/python_api.py

```diff
@@ -44,6 +52,38 @@ def build_docker(
 
     See https://www.mlflow.org/docs/latest/python_api/mlflow.pyfunc.html for more information on the
     'python_function' flavor.
+
+
+    :param model_uri: URI to the model. A local path, a 'runs:/' URI, or a remote storage URI (e.g.,
```

Comment:
should we convert to Google docstring format? it looks like in this file, only this function and `predict` have docstrings. it might make it easier on @michael-berk haha

Reply:
Wow true will convert to the new style!